backends/arm/test/tester/arm_tester.py

```diff
@@ -41,6 +41,33 @@ class ArmBackendSelector(Enum):
     ETHOS_U55 = "ethos-u55"
 
 
+class Partition(Partition):
+    def dump_artifact(self, path_to_dump: Optional[str]):
+        super().dump_artifact(path_to_dump)
+        from pprint import pformat
+
+        to_print = None
+        for spec in self.graph_module.lowered_module_0.compile_specs:
```

Comment:
Is there an API to or other 'nicer' way to access

`graph_module.lowered_module_0.compile_specs`

and

`graph_module.lowered_module_0.processed_bytes`

Reply:
Not that I am aware of, cc @cccclai